tests/regression_tests/rotation/test.py

```diff
@@ -1,6 +1,40 @@
-from tests.testing_harness import TestHarness
+import openmc
```

Comment:
was this file changed on purpose? seems unrelated to the DAGMC based changes? maybe im wrong?

Reply:
Ah, no you're right. This is the result of a mishandled rebase conflict. I'll sort it out.